JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/scikitwrapper/ScikitLearnWrapperCommandBuilder.java

```diff
@@ -202,6 +212,9 @@ protected void checkRequirementsTrainTestMode() {
 			processParameters.addAll(Arrays.asList(PREDICT_OUTPUT_FLAG, this.predictOutputFile));
 		}
 		processParameters.addAll(Arrays.asList(SEED_FLAG, String.valueOf(this.seed)));
+		if (this.targetIndices != null && this.targetIndices.length > 0) {
```

Comment:
what happens if we have no targets given?

Reply:
This is handeled as usual: the last index is taken